tree.go

```diff
@@ -873,6 +873,7 @@ func (n *InternalNode) GetProofItems(keys keylist, resolver NodeResolverFn) (*Pr
 				if err != nil {
 					return nil, nil, nil, err
 				}
+				n.children[i] = c
```

Comment:
If we don't do this, the line below `pec, es, other, err := n.children[childIdx].GetProofItems(group, resolver)` (L930) would fail since `n.children[childrenIdx]` can be a `HashedNode` and `GetProofItems` isn't allowed.